mgr/src/server/plugins/volume_heal.cr

```diff
@@ -104,9 +104,8 @@ post "/api/v1/pools/:pool_name/volumes/:volume_name/heal/start" do |env|
   client_volfile = "/var/lib/kadalu/volfiles/client-dev-#{volume_name}.vol"
```

Comment:
@aravindavk Shouldn't this be `volume_name.vol` for nfs-ganesha to mount. ?

Reply:
It should be fine since volfile path is used instead of volfile server. (Line 114).

Mount volfiles are anyway different even though client template is reused here.